src/components/css-components/z-link/styles.css

```diff
@@ -25,150 +42,143 @@ button.z-link:active {
-  pointer-events: none;
-  text-decoration: none;
+a.z-link.z-link-blue,
+button.z-link.z-link-blue {
+  color: var(--color-text-link-blue);
+  fill: var(--color-text-link-blue);
 }
 
-a.z-link.z-link-underline,
-button.z-link.z-link-underline {
-  text-decoration: underline;
+a.z-link.z-link-red,
+button.z-link.z-link-red {
+  color: var(--color-text-link-red);
+  fill: var(--color-text-link-red);
 }
 
 a.z-link.z-link-inverse,
 button.z-link.z-link-inverse {
-  color: var(--color-inverse-link);
-  fill: var(--color-inverse-link);
+  color: var(--color-text-link-inverse);
+  fill: var(--color-text-link-inverse);
 }
 
-a.z-link.z-link-inverse:hover,
-button.z-link.z-link-inverse:hover {
-  color: var(--color-inverse-hover-link);
-  fill: var(--color-inverse-hover-link);
+a.z-link.z-link-blue.z-link-active,
+button.z-link.z-link-blue.z-link-active {
+  color: var(--color-text-link-blue);
+  fill: var(--color-text-link-blue);
 }
 
-a.z-link.z-link-inverse:active,
-button.z-link.z-link-inverse:active,
-a.z-link.z-link-inverse:focus,
-button.z-link.z-link-inverse:focus {
-  color: var(--color-inverse-pressed-link);
-  fill: var(--color-inverse-pressed-link);
+a.z-link.z-link-red.z-link-active,
+button.z-link.z-link-red.z-link-active {
+  color: var(--color-text-link-red);
+  fill: var(--color-text-link-red);
 }
 
-a.z-link.z-link-inverse:visited,
-button.z-link.z-link-inverse:visited {
-  color: var(--color-inverse-visited-link);
-  fill: var(--color-inverse-visited-link);
+a.z-link.z-link-inverse.z-link-active,
+button.z-link.z-link-inverse.z-link-active {
+  color: var(--color-text-link-inverse);
+  fill: var(--color-text-link-inverse);
 }
 
-a.z-link.z-link-black,
-button.z-link.z-link-black {
-  color: var(--gray800);
-  fill: var(--gray800);
+a.z-link.z-link-blue:hover,
+button.z-link.z-link-blue:hover {
+  color: var(--color-text-link-blue-hover);
+  fill: var(--color-text-link-blue-hover);
 }
 
-a.z-link.z-link-black:hover:visited,
-button.z-link.z-link-black:hover:visited {
-  color: var(--gray800);
-  fill: var(--gray800);
+a.z-link.z-link-red:hover,
+button.z-link.z-link-red:hover {
+  color: var(--color-text-link-red-hover);
+  fill: var(--color-text-link-red-hover);
 }
 
-a.z-link.z-link-black:hover,
-button.z-link.z-link-black:hover,
-a.z-link.z-link-black.z-link-active:focus,
-button.z-link.z-link-black.z-link-active:focus {
-  color: var(--color-black);
-  fill: var(--color-black);
+a.z-link.z-link-inverse:hover,
+button.z-link.z-link-inverse:hover {
+  color: var(--color-text-link-inverse);
+  fill: var(--color-text-link-inverse);
 }
 
-a.z-link.z-link-black:active,
-button.z-link.z-link-black:active,
-a.z-link.z-link-black.z-link-active:active,
-button.z-link.z-link-black.z-link-active:active {
-  color: var(--gray700);
-  fill: var(--gray700);
+a.z-link.z-link-blue:active,
+button.z-link.z-link-blue:active,
+a.z-link.z-link-blue:focus,
+button.z-link.z-link-blue:focus {
+  color: var(--color-text-link-blue-active);
```

Comment:
Io forse raggrupperei tutte le parti in cui sovrascriviamo il default.

Reply:
Ho sbagliato il punto in cui fare questo commento: intendevo per la parte tra riga 62 e 124.